misk-action-scopes/src/main/kotlin/misk/scope/ActionScope.kt

```diff
@@ -25,7 +25,7 @@ class ActionScope @Inject internal constructor(
 
   /** Starts the scope on a thread with the provided seed data */
   fun enter(seedData: Map<Key<*>, Any?>): ActionScope {
-    check(tls.get() == null) {
+    check(!inScope()) {
```

Comment:
These changes are functionally equivalent. I just replaced it so that it's easier to read and understand.